backend/grades/resource/formatted/fall_2022.csv

```diff
@@ -0,0 +1,3086 @@
+ESPM,H175A,26936,001,Sen Res Sem Env Sci,Patina Mendez; Danielle Perryman; Robin Lopez,under,0,34,3,1,1,0,0,0,0,0,0,0,0,0,0
+ESPM,H175L,26898,001,Sen Res Lab Env Sci,Patina Mendez,under,0,26,2,0,0,0,0,0,0,0,0,0,0,0,0
+ETH STD,101B,24678,001,Humanities Methods,Gregory Choy,under,6,42,1,2,3,0,0,0,0,0,0,0,1,2,0
+ETH STD,11AC,24674,001,Intro To Eth Std,Juana Maria Rodriguez; Sofia Chavez; Pilar Jefferson,under,5,77,11,0,4,0,2,1,3,0,0,0,1,11,0
+ETH STD,12,33315,001,Contemp Issues Es,Catherine Choy; Fernanda Rivera,under,15,9,1,2,1,0,0,0,1,0,0,0,0,0,0
+ETH STD,130,21013,001,Mex Chic Art Hist,Jesus Barraza,under,23,3,0,1,0,0,0,0,0,0,0,0,0,3,0
+ETH STD,132,33749,001,Day Of The Dead,Jesus Barraza,under,28,5,1,0,1,0,0,0,0,0,0,0,0,0,0
+ETH STD,135B,23513,001,Contemp Latino Film,Raymond Telles; Peter Kim,under,3,35,32,7,7,1,0,0,1,0,0,0,0,4,1
+ETH STD,150,21440,001,Mix Racial Descent,Gregory Choy,under,0,29,0,3,0,0,0,0,0,0,0,0,0,1,0
+ETH STD,159,21011,001,Mexican Immigration,Pablo Gonzalez,under,5,31,9,1,7,0,0,0,0,0,0,0,0,0,0
+ETH STD,161,23504,001,Central Amer People,Enrique Lima,under,0,13,6,1,1,0,0,0,0,0,0,0,0,0,0
+ETH STD,173AC,23457,001,Indig Ppls Gbl Ineq,Thomas Biolsi; Lucy Gill,under,5,6,7,2,0,1,0,0,0,0,0,0,0,0,0
+ETH STD,175,32341,001,Latinx And The Env,Federico Castillo,under,4,19,7,2,4,0,0,0,1,0,0,0,0,0,0
+ETH STD,175,21441,001,Lit From Eth Mvmts,Gregory Choy,under,0,30,0,2,1,0,0,0,0,0,0,0,0,1,0
+ETH STD,180,23856,002,Comp Eth Std Topics,John Dougherty,under,1,16,1,0,0,0,0,0,0,0,0,0,1,0,0
+ETH STD,180,31042,003,Comp Eth Std Topics,Philip Ting,under,0,13,4,0,0,0,0,0,0,0,0,0,0,2,0
+ETH STD,190,25962,001,Adv Sem Chx/Lx Std,Juan Berumen,under,0,2,6,4,2,2,2,1,1,0,1,0,1,1,0
+ETH STD,194,24049,001,Stud Acad Success,Luisa Giulianetti; Jan Carmelo Bautista,under,4,11,0,1,4,0,0,0,0,0,0,0,1,0,0
+ETH STD,197,16370,014,Field Study,Luisa Giulianetti; Pablo Gonzalez; Andrew Henry,under,0,0,0,0,0,0,0,0,0,0,0,0,0,26,1
+ETH STD,197,16375,012,Field Study,Luisa Giulianetti; Pablo Gonzalez,under,0,0,0,0,0,0,0,0,0,0,0,0,0,11,1
+ETH STD,197,16380,005,Field Study,Victoria Robinson; Aileen Pagdanganan,under,0,0,0,0,0,0,0,0,0,0,0,0,0,26,1
+ETH STD,197,16384,009,Field Study,Laura Jimenez-Olvera; Dewey St Germaine; Keith Feldman,under,0,0,0,0,0,0,0,0,0,0,0,0,0,42,1
+ETH STD,197,16386,011,Field Study,Luisa Giulianetti; Pablo Gonzalez,under,0,0,0,0,0,0,0,0,0,0,0,0,0,21,1
+ETH STD,197,17376,001,Field Study,Laura Jimenez-Olvera; Pablo Gonzalez,under,0,0,0,0,0,0,0,0,0,0,0,0,0,27,1
+ETH STD,198,21052,008,Directed Group Stdy,Maria Gallegos-Diaz; Laura Jimenez-Olvera; Juan Berumen,under,0,0,0,0,0,0,0,0,0,0,0,0,0,26,1
+ETH STD,198,23302,008,Superv Group Study,Ronald Williams; Keith Feldman,under,0,0,0,0,0,0,0,0,0,0,0,0,0,17,2
+ETH STD,39,24421,001,Fresh/Soph Seminar,Christian Paiz,under,11,1,0,0,1,0,0,0,1,0,0,0,0,1,0
+ETH STD,39,33748,001,Fresh/Soph Seminar,Christian Paiz,under,16,0,0,0,0,0,0,0,0,0,0,0,0,1,0

```

Comment:
In order for the change to the abbreviation to be reflected in the formatted csv, the script will need to be run again.